src/core/qgsrenderedfeaturehandlerinterface.h

```diff
@@ -50,10 +50,19 @@ class CORE_EXPORT QgsRenderedFeatureHandlerInterface
 
     struct CORE_EXPORT RenderedFeatureContext
     {
-      ///@cond PRIVATE
-      // required to allow compilation only until real members are present
-      bool dummy; SIP_SKIP
-      ///@endcond
+
+      /**
+       * Constructor for RenderedFeatureContext.
+       * \param renderContext The render context which was used while rendering feature.
```

Comment:
I assume the pointer here isn't made to have its ownership transferred?

Reply:
I swapped it to a const reference for clarity